core/audio_unsupported.go

```diff
@@ -0,0 +1,36 @@
+//+build netgo windows android
+
+package core
+
+import (
+	"log"
+
+	"engo.io/ecs"
+)
+
+const (
+	defaultHeightModifier float32 = 1
+)
+
+var MasterVolume float64 = 1
+
+// AudioComponent is a Component which is used by the AudioSystem
+type AudioComponent struct {
+	File       string
+	Repeat     bool
+	Background bool
+	RawVolume  float64
+}
+
+// AudioSystem is a System that allows for sound effects and / or music
+type AudioSystem struct {
+	HeightModifier float32
+}
+
+func (as *AudioSystem) New(*ecs.World) {
+	log.Println("Warning: audio is not yet implemented on this platform")
+}
+
+func (as *AudioSystem) Add(*ecs.BasicEntity, *AudioComponent, *SpaceComponent) {}
+func (as *AudioSystem) Remove(basic ecs.BasicEntity)                           {}
+func (as *AudioSystem) Update(dt float32)                                      {}
```

Comment:
I basically removed the `engo_js` and `engo_windows` methods. It basically says either "unsupported", or it's the default file which (in the future) will use `engo.io/audio`.